### ISSUE TITLE
Multiply -1 before loss.backward()?

### ISSUE THREAD

**Issue by nijianmo** [kmkurn/pytorch-crf#27]
Hi, thanks for providing this great tool!

I have one quick question about the loglikelihood returned by forward function: 
loss = model(emissions, tags, mask=mask).

Is the returned value the negative loglikelihood or likelihood? Should I multiply -1 before loss.backward()?

Thanks!

**Comment by kmkurn** [comment]
Hi, thank you for using this library!

The returned value is the log likelihood. So, to use it as a loss objective to minimize, you'd want to multiply it by -1 first. Hope this helps!

**Comment by kmkurn** [comment]
@nijianmo It seems that you've found your answer, so I'm closing this issue.
